scripts/deploy-release.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Variables
+DISTRIBUTION_ID="E1GEDS049Y53CA"
+FILE_PATH="dist/capture-eye.bundled.js"
+S3_BUCKET="numbers-static"
+
+# Define S3 keys for paths
+LATEST_KEY="capture-eye/release/latest/capture-eye.bundled.js"
+VERSION_KEY="capture-eye/release/$(npm version --no-git-tag-version patch)/capture-eye.bundled.js"
+
+# CloudFront invalidation paths
+INVALIDATION_PATHS=(
+  "/capture-eye/release/latest/capture-eye.bundled.js"
+  "/capture-eye/release/$(npm version --no-git-tag-version patch)/capture-eye.bundled.js"
```

Comment:
do you want to retrieve the package version instead of updating it?
Also, maybe you can reuse the variables LATEST_KEY and VERSION_KEY above.